js/atoms/src/Info.svelte

```diff
@@ -9,11 +9,11 @@
 </div>
 
 <style>
-	div {
+	:global(.md) {
 		margin-bottom: var(--spacing-lg);
 		color: var(--block-info-text-color);
-		font-weight: var(--block-info-text-weight);
-		font-size: var(--block-info-text-size);
+		font-weight: var(--block-info-text-weight) !important;
+		font-size: var(--block-info-text-size) !important;
 		line-height: var(--line-sm);
```

Comment:
My understanding is that its not good practice to use the `!important` here. It would be better to pass in an optional parameter into `Markdown` that adjusts the font size to be smaller

Reply:
Hmm yeah probably not, but the actual font size that I'm overriding is not in the `Markdown` component but even lower in the stack in the Embed component. Font size would have to be passed a few time I think, which is probably messy. We use `!important` is a bunch of other places too.

Reply:
Got it

Reply:
Let's avoid introducing !important when easily possible, its a bad practice.